### ISSUE TITLE
Error compiling tests with mockery

### ISSUE THREAD

**Issue by jmckitrick** [igrishaev/mockery#4]
I'm seeing this error:

var: #'mockery.core/make-mock is not public

while compiling this code:

```clojure
(deftest login-test
  (testing "TEST"
    (mock/with-mocks
      [post {:target :post
             :return nil}
       get {:target :get
            :return nil}]
      (login)
      (is (= @post
             {}))
      (is (= @get
             {})))))
```

**Comment by igrishaev** [comment]
thank you

**Comment by igrishaev** [comment]
@j-mckitrick fixed in `[mockery "0.1.1"]`
